DeveloperGuide.md

```diff
@@ -0,0 +1,408 @@
+# glTF Universal Textures Developer Guide
```

Comment:
Is this still the right title, if we are moving away from the term "Universal Textures"? Suggestion:

```suggestion
# KTX 2.0 / Basis Universal Textures in glTF — Developer Guide
```

Reply:
Good catch. By the way, this document doesn't mention glTF except in the title.

Reply:
The KTX section (https://github.com/KhronosGroup/3D-Formats-Guidelines/pull/2) would add another minor reference to glTF. I think that's fine, and we don't need more than that except perhaps a link to `KHR_texture_basisu`. Or would you prefer to remove these references?

Reply:
This guide does not depend on glTF and can be used on its own. Still, there's a value in referring to the glTF extension that enables the integration.

So let's remove the reference from the title and add a reference to the extension.

Reply:
Sounds good to me.